scenarioo-client/app/sketcher/stepSketch.controller.js

```diff
@@ -107,7 +107,7 @@ function StepSketchController($scope, $routeParams, $location, HostnameAndPort,
         }
 
         var selected = SelectedBranchAndBuildService.selected();
-        return HostnameAndPort.forLink() + 'rest/branch/' + selected.branch + '/issue/' + issueId + '/scenariosketch/' + scenarioSketchId + '/stepsketch/' + stepSketchId + '/image/original.png';
+        return 'rest/branch/' + selected.branch + '/issue/' + issueId + '/scenariosketch/' + scenarioSketchId + '/stepsketch/' + stepSketchId + '/image/original.png';
```

Comment:
Won't angular do that automatically? It's used like this:
{{vm.getOriginalScreenshotUrl()}}

Reply:
I think angular would only do it automatically, if we used usual angular way to put parameters into URLs - but since we use string concatenation here directly, I think it does not do it. But maybe I am wrong. I have just seen that in other places we encode branch name before putting it into URLs and It hink it would be better to do that consistently everywhere!